3rdParty/StormLib/src/StormPort.h

```diff
@@ -360,6 +362,6 @@
 //
 
 #define bool int
-extern "C" void TranslateFileName(char* dst, int dstLen, const char* src);

```

Comment:
Is this something we already added to StormLib?

Reply:
I'm not sure. Where does this StormLib come from? What needs to be done here?

Reply:
https://github.com/ladislav-zezula/StormLib We hope to start upstreaming some of our changes as well as syncing the latest changes from them.

Reply:
Doesn't seem to exist upstream: https://github.com/ladislav-zezula/StormLib/search?q=TranslateFileName&unscoped_q=TranslateFileName